conda_vendor/conda_vendor.py

```diff
@@ -470,9 +506,9 @@ def yaml_dump_ironbank_manifest(package_list: List[FetchAction]):
 
         resources["resources"].append(resource)
 
-    yaml = YAML()
+    yaml_ = YAML()
```

Comment:
Is there some semantic reason for the trailing underscore?

Reply:
to not shadow import yaml at the top.  it was to silence a pylint warning.

Reply:
makes sense